sound/soc/sof/intel/hda.h

```diff
@@ -376,11 +340,18 @@ struct sof_intel_hda_dev {
 	const struct sof_intel_dsp_desc *desc;
 
 	/*trace */
-	struct sof_intel_hda_stream *dtrace_stream;
+	struct hdac_ext_stream *dtrace_stream;
 
 	int irq;
 };
 
+#define SOF_STREAM_SD_OFFSET(s) (0x20 * ((s)->index) + SOF_HDA_ADSP_LOADER_BASE)
```

Comment:
where does 0x20 come from ?

Reply:
we should change it to an macro, it is size of all registers belong to a separated stream.